src/keys.cc

```diff
@@ -182,16 +200,39 @@ UnitTest test_keys{[]()
     KeyList keys{
          { ' ' },
          { 'c' },
+         { Key::Up },
          { Key::Modifiers::Alt, 'j' },
-         { Key::Modifiers::Control, 'r' }
+         { Key::Modifiers::Control, 'r' },
```

Comment:
can you convert those to `alt('j'), ctrl('r'), and `shift(Key::Up)` while your are at it ?